src/devices/Si7021/Resolution.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Si7021
+{
+    /// <summary>
+    /// Si7021 Measurement Resolution
+    /// </summary>
+    public enum Resolution : byte
+    {
+        /// <summary>
+        /// Humidity 12-bit, Temperature 14-bit
+        /// </summary>
+        Resolution1 = 0b00,
```

Comment:
I wish the only useful name I can think of wasn't super long 😄

Reply:
Actually, I haven't come up with any suitable name. This Resolution can not be simply described by high and low.😂